app/controllers/auth_controller.rb

```diff
@@ -6,6 +6,8 @@ def create
       auth_with_okta
     elsif data_attrs[:facebook_access_token]
       auth_with_facebook
+    elsif data_attrs[:google_access_token]
```

Comment:
are you using an `access_token` or an `id_token` for google authentication?

Reply:
because we are using an `id_token` to authenticate for google, this attribute should be `google_id_token`